README.md

```diff
@@ -1,83 +1,75 @@
-## Kubecost
+<img src="./opencost-header.png"/>
 
-Kubecost models give teams visibility into current and historical Kubernetes spend and resource allocation. These models  provide cost transparency in Kubernetes environments that support multiple applications, teams, departments, etc.
+# OpenCost — your favorite open source cost monitoring tool for Kubernetes
 
-![Kubecost allocation UI](/allocation-drilldown.gif)
+OpenCost models give teams visibility into current and historical Kubernetes spend and resource allocation. These models provide cost transparency in Kubernetes environments that support multiple applications, teams, departments, etc.
+
+OpenCost is developed by [Kubecost](https://kubecost.com).
```

Comment:
Should we link to our intentions here? https://github.com/kubecost/cost-model/issues/1224